test/cmdlineTests/viair_subobject_optimization/output

```diff
@@ -134,7 +126,9 @@ sub_0: assembly {
       tag_4
       jumpi
       pop
-      jump(tag_3)
+      0x00
```

Comment:
Why does it not deduplicate anymore?

Reply:
Not entirely sure.
But I find it suspicious that the evmasm optimiser first inlines, then removes unused tags and then deduplicates (which relies on stuff being tagged, if I read it correctly)... I guess there's something off with that there.

Reply:
That is true. I think the deduplicator even ignores additional tags when it compares for equality.